docker/build_scripts/install-runtime-packages.sh

```diff
@@ -28,6 +31,8 @@ if [ "${AUDITWHEEL_POLICY}" == "manylinux2010" ] || [ "${AUDITWHEEL_POLICY}" ==
 	MANYLINUX_DEPS="glibc-devel libstdc++-devel glib2-devel libX11-devel libXext-devel libXrender-devel mesa-libGL-devel libICE-devel libSM-devel"
 elif [ "${AUDITWHEEL_POLICY}" == "manylinux_2_24" ]; then
 	MANYLINUX_DEPS="libc6-dev libstdc++-6-dev libglib2.0-dev libx11-dev libxext-dev libxrender-dev libgl1-mesa-dev libice-dev libsm-dev"
+elif [ "${AUDITWHEEL_POLICY}" == "musllinux_1_1" ]; then
+	MANYLINUX_DEPS="musl-dev libstdc++ glib-dev libx11-dev libxext-dev libxrender-dev mesa-dev libice-dev libsm-dev"
```

Comment:
I tried compiling a simple `c++`/`g++` hello world with this image.

It looks like `g++` is not fully set up yet:
```console
$ c++ main.cpp
ld: cannot find -lm
collect2: error: ld returned 1 exit status
```

I searched for a `libm` in the `/` dir and cannot find one. Adding `-static` also does not solve the problem.

Is something still missing in this base image? :)

Reply:
Everything works now! I think my problem was about a week old.